### ISSUE TITLE
gphotoframe disappears when it is iconified

### ISSUE THREAD

**Issue by GoogleCodeExporter** [yendo/gphotoframe#5]
```
What steps will reproduce the problem?
1. start gphotoframe
2. click on the desktop icon to hide all windows from the desktop
3. gphotoframe is lost

What is the expected output? What do you see instead?
I'd like gphotoframe to stay on the desktop

What version of the product are you using? On what operating system?
I am using ubuntu 8.04 and gphotoframe 0.1

Please provide any additional information below.


```

Original issue reported on code.google.com by `K.J.Nies...@gmail.com` on 5 May 2009 at 9:14

**Comment by GoogleCodeExporter** [comment]
```
I will implement:

1) If the window is iconified, the skip_task_bar will be disabled 
2) if the window is fixed on the root window (issue #1), it will not be 
iconified.
```

Original comment by `yendo0206` on 5 May 2009 at 11:05
- Changed title: **gphotoframe disappears when it is iconified**
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
> 1) If the window is iconified, the skip_task_bar will be disabled 

Implemented in r40.

> 2) if the window is fixed on the root window ( issue #1 ), it will not be 
iconified.

Implemented in r45.

```

Original comment by `yendo0206` on 8 May 2009 at 1:27
- Changed state: **Fixed**
